doc/classes/Particles2D.xml

```diff
@@ -39,8 +40,10 @@
 			How rapidly particles in an emission cycle are emitted. If greater than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+			This setting can be used to set the particle system to render at a fixed FPS. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the particle system itself.
 		</member>
 		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta">
+			This can be used to turn Fract Delta on or off.
```

Comment:
It would be better to explain what Fract Delta is :)
It's quite logical that a boolean named `fract_delta` is used to toggle fract delta, but that doesn't tell us what it actually does. It would also be good to mention what the default value is as done for other properties.

Reply:
Updated the definitions to the capture_rect() and modified the fract_delta definition as well. Also included the default value for the parameter like the rest.